service/core.go

```diff
@@ -126,25 +126,25 @@ func (s *serviceCore) AuthClient() auth.Client {
 }
 
 func (s *serviceCore) Name() string {
```

Comment:
It's not 100% clear that application ID maps back to the service Name. Did you consider renaming the yaml tags to service instead of renaming the config attribute names to application?